src/syntax.ml

```diff
@@ -262,18 +263,19 @@ let eq_p_cmd : p_cmd eq = fun c1 c2 ->
       m1 = m2 && eq_require_mode r1 r2
   | (P_open(m1)                  , P_open(m2)                  ) ->
       m1 = m2
-  | (P_symbol(l1,s1,a1)          , P_symbol(l2,s2,a2)          ) ->
-      l1 = l2 && eq_ident s1 s2 && eq_p_term a1 a2
+  | (P_symbol(l1,s1,al1,a1)      , P_symbol(l2,s2,al2,a2)      ) ->
+     l1 = l2 && eq_ident s1 s2 && eq_p_term a1 a2
```

Comment:
Indentation is not correct here, missing one space before `l1 = l2 && ...` and on the following line.

Reply:
Note that both `ocp-indent` and `ocamlformat` choose instead to remove the extra before `l1 = l2` [2 minus than what you use]. This style makes sense as you are 4 spaces right of the `|`, and helps save very precious horizontal space.

Reply:
I know that some people do that, but I don't like this style since it hurts readability in my opinion. I largely prefer having well-identified block, and I rely on other methods to shorten the length of lines (avoiding too many levels of nesting, ...).